Makefile.js

```diff
@@ -61,7 +61,7 @@ const NODE = "node ", // intentional extra space
 
     // Utilities - intentional extra space at the end of each string
     MOCHA = `${NODE_MODULES}mocha/bin/_mocha `,
-    ESLINT = `${NODE} bin/eslint.js --report-unused-disable-directives `,
+    ESLINT = `${NODE} bin/eslint.js `,
```

Comment:
```suggestion
    ESLINT = `${NODE} bin/eslint.js --report-unused-disable-directives `,
```

We should keep the CLI option because it produces `error` severity. The configuration option produces warnings, so the linting will pass with exit code 0.

I think it still makes sense to have both - the configuration option can find the error earlier (in IDE).

Reply:
ah, didn't noticed that! I think it makes sense to allowit configurable in eslintrc:

```js
reportUnusedDisableDirectives: "error"/"warn"/"off"
```

for backward compat: true => "warn", false => "off". thoughts?

Reply:
This was discussed in https://github.com/eslint/rfcs/pull/22, and it's intentionally only `true`/`false` (translates to `"warn"`/`"off"`) because of our semver policy, which says that bug fixes that produce fewer errors should not break builds. 

Fewer errors from a rule might actually produce more errors for users that had an `eslint-disable` comment on the false positive and  `reportUnusedDisableDirectives` configured to `"error"`.

CLI flags like `--report-unused-disable-directives` and `--max-warnings` are considered as opt-in in this regard.

Reply:
I see, thanks for the explanation! let's update the eslint-config-eslint, so it can be applied to other repos in eslint org.